subprojects/platform-native/src/testFixtures/groovy/org/gradle/nativeplatform/fixtures/RequiresInstalledToolChain.groovy

```diff
@@ -18,12 +18,14 @@ package org.gradle.nativeplatform.fixtures
 import org.spockframework.runtime.extension.ExtensionAnnotation
 
 import java.lang.annotation.ElementType
+import java.lang.annotation.Inherited
 import java.lang.annotation.Retention
 import java.lang.annotation.RetentionPolicy
 import java.lang.annotation.Target
 
 @Retention(RetentionPolicy.RUNTIME)
 @Target([ElementType.METHOD, ElementType.TYPE])
+@Inherited
```

Comment:
I feel this should be present (same as `@Requires`), however, there may be a good reason why this wasn't present before.